Shocksupershaman - Copy/src/main/java/com/example/shocksupershaman/shocktask1/UserService.java

```diff
@@ -0,0 +1,42 @@
+package com.example.shocksupershaman.shocktask1;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.*;
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.stream.Collectors;
+
+@Data
+@NoArgsConstructor
+@Service

```

Comment:
We don't need  Data , NoArgsConstructor for service type code. @Service is enough